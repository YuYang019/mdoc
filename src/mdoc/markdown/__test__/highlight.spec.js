const { Md } = require('./util')
const { getFragment } = require('../../test-utils')
const highlight = require('../lib/highlight')

const md = Md()
const mdH = Md().set({ highlight })

describe('highlight', () => {
  test('should highlight the code', done => {
    const input = getFragment(__dirname, 'code.md')
    const output1 = md.render(input)
    const output2 = mdH.render(input)
    expect(output1 === output2).toBe(false)
    expect(output2).toMatchSnapshot()
    done()
  })
})
