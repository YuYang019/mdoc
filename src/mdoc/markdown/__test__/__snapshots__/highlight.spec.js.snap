// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`highlight should highlight the code 1`] = `
"<pre class=\\"language-js\\"><code>mdoc dev <span class=\\"token punctuation\\">.</span>
</code></pre>
"
`;
